tests/flamegraph-nameattr.rs

```diff
@@ -0,0 +1,55 @@
+#[macro_use]
+extern crate pretty_assertions;
+
+extern crate inferno;
+
+use inferno::flamegraph;
+use std::fs::File;
+use std::io::{BufRead, BufReader, Cursor};
+use std::path::PathBuf;
+
+#[test]
+fn flamegraph_nameattr() {
+    let input_file = "./flamegraph/test/results/perf-cycles-instructions-01-collapsed-all.txt";
+    let expected_result_file = "./tests/data/nameattr/nameattr.svg";
```

Comment:
I think you forgot to add this file? It could be that you need to change the line
```
*.svg
```
in `.gitignore` to read
```
/*.svg
```
instead first.

Reply:
Oops!